unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -348,8 +337,8 @@ public DecentralandEntity CreateEntity(string id)
 
             newEntity.OnCleanupEvent += po.OnCleanup;
 
-            if (useBoundariesChecker || SceneController.i.isDebugMode)
-                newEntity.OnShapeUpdated += boundariesChecker.EvaluateEntityPosition;
+            if (SceneController.i.useBoundariesChecker)
```

Comment:
Maybe its a bit more accurate to null-check `SceneController.i.boundariesChecker` instead of this bool. If the bool changes purpose or the class isn't initialized correctly the nullity can slip away.

Reply:
> Maybe its a bit more accurate to null-check `SceneController.i.boundariesChecker` instead of this bool. If the bool changes purpose or the class isn't initialized correctly the nullity can slip away.

This suggestion may arise some issues with builder. I think that builder's will need to set SceneController.i.useBoundariesChecker to false (for example to avoid adding entities to the list of entities to check, cause builder do the boundaries check itself) but still need that the instance of boundariesChecker exist

Reply:
That seems like an antipattern, why would the builder need such a confusing setup?

Reply:
Ok, after discussing this personally we decided to move this suggestion to the future.